src/components/NavBar/style.tsx

```diff
@@ -0,0 +1,83 @@
+import styled from 'styled-components';
+import COLORS from '@/styles/colors';
+import { openSans } from '@/styles/fonts';
+import { LinkColored } from '@/styles/text';
+
+export const NavBarContainer = styled.div`
+  /* top: 0;
+  left: 0;
+  position: absolute; */
+  width: 100%;
+  height: 78px;
```

Comment:
do we want to change this to rem? i remember @jinkang-0 said that it's best practice to change pixels to rem, especially when something is more than 5 pixels?

Reply:
over break jinkang told me something about how maybe not everything should be in rem, maybe let's add this to the agenda for tomorrow's worksesh so we're all on the same page with best styling practices!